Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
+     * will scroll with the content at the top of the section until it reaches
+     * the top of the screen, at which point it will stick to the top until it
+     * is pushed off the screen by the next section header.
+     * @platform ios
+     */
+    stickySectionHeaders: React.PropTypes.bool,
```

Comment:
What's the purpose of extra flag? I believe it's possible to check `stickyHeaderIndices` length instead. something like 
```js
if (stickyHeaderIndices.length) {
  stickyHeaderIndices.concat(sectionHeaderIndices);
}
```

Does it make any sense?